lib/crypto/password.js

```diff
@@ -51,7 +51,8 @@ module.exports = function(log, config) {
         return hkdf(stretched, context, null, 32)
           .then(
             function (wrapper) {
-              return butil.xorBuffers(wrapper, wrapped)
+              return butil.xorBuffers(wrapper, Buffer.from(wrapped, 'hex'))
```

Comment:
Looks like `xorBuffers` already bufferizes both arguments above, so you can pass in `wrapped` unadulterated?